backend/backend/urls.py

```diff
@@ -46,6 +51,14 @@
 	path('friends/<int:friend_id>', FriendshipDetailView.as_view()),
 
 	# Tokens views
-	path('authenticate-guest-user/', GuestUserAuthenticationView.as_view()),
-	path('remove-guest-user/', DeactivateGuestUserTokenView.as_view()),
+	path('authenticate-guest-user-for-single-match/', SingleMatchGuestUserAuthenticationView.as_view()),
+	path('deactivate-match-token/', DeactivateMatchTokenView.as_view()),
+
+	# Match Views
+	path('start-single-match/', LaunchSingleMatchView.as_view()),
+	path('start-test-match/', LaunchTestMatchView.as_view()),
```

Comment:
I think these routes are .... well weird :P To think about this Rest API we are building in a way that it would be build in a consistent way that thinking of a new / external client starting to use it, we should use the convention set previously. The new client should be able to understand what exactly does this route call do, what kind of data does it update / change?

Are these routes not for POSTing a match? or GETing a launched match? so wouldnt something like GET /match/ or POST /match/ or even POST /match/start would make it easier to dynamically also use the conventions we already have in the frontend too to use these routes?

Reply:
I understand what you mean. Those are placeholders as I haven't started on the tournament api yet so I'm not sure how the endpoints for that will work yet.

I could see there being stuff like /tournament/authenticate-guest, in which case I could see single matches having stuff like /single-match/authenticate-guest and /single-match/start-match, or instead of single-match you'd have something like /lobby/etc.

Reply:
this is basically the same issue as with the column names: we need to start having conventions on how and what we mean when we say this or that. One route to get users cant be /users/ and adding one POST /users/ and then updating a user would out of nowhere be PUT /update-user/. Please apply the existing convention! Here is what I suggested in the Discord:
_'can you change the token routes to follow the convetion of other routes? POSTin a token should be something like POST /tokens/ or /tokens/guest/ and the inactivation route should be PUT method and either the same as the other one, /tokens/ or /tokens/guest/'_

Reply:
for the match, it could be something like /match/?command=start for example, like the friends route

Reply:
I am with Meri on that. Fully understandable that you had to navigate that in your own and did the best you knew how, but after some time away and given the time line we might want to step it up and set up things properly from the get go. I will approve the PR now and create an issue. I leave it up to you to fix it either now or on the new issue :D

Reply:
I'll make a new issue for this!